### DISCUSSION TITLE
Search for user edits and UI consistency

### DISCUSSION THREAD

**Discussion by comradekingu** [WeblateOrg/weblate#5386] [Ideas]
Clicking https://hosted.weblate.org/user/kingu/#changes gives 6 entries.
Then "see all" gives 10 more, and a search.

_It seems possible to save the user the "see all" click, because almost always 6 entries isn't going to do it, or the user is looking for the search to begin with._

I would obviously like the 16 _and_ the search right away, but that isn't good for small screens, and figuring out actual screen estate is hard, so how about_

6 entries and the search, with paginator that can show 5, 10 or 20 entries at once.

the search is up top, so why not also the user edit search up top?

I like things being in the same place every time, like
In https://hosted.weblate.org/projects/remmina/#search and
https://hosted.weblate.org/zen/remmina/remmina/nb_NO/?offset=1&q=install&sort_by=-priority%2Cposition&checksum=

whereas "import" in https://hosted.weblate.org/memory/ is also all the way at the bottom.

The alternative to configuring the amount of hits in the amount of hits is to use
"number of strings nearby" in https://hosted.weblate.org/accounts/profile/#preferences

by calling it "Vertical element limit" with the tooltip being 
"number entries in searches and nearby strings to show in the full editor"

_But having this option in there was always worse than editing it where it is needed._

Otherwise maybe the search could be in https://hosted.weblate.org/user/kingu/#activity?
Or the search could be its own tab, or someone has a good idea.

**Comment by nijel** [comment]
> Clicking https://hosted.weblate.org/user/kingu/#changes gives 6 entries.

It gives 10 entries:

https://github.com/WeblateOrg/weblate/blob/0b23f9d84315f3f566b6c43e075c64e7cb2e8752/weblate/accounts/views.py#L574-L575

The search on changes page is on the bottom because the form is huge. Once we can make it one line as in other locations, it would be in the top.
